docs/knowledgebase/integrate/subkey.md

```diff
@@ -291,138 +281,179 @@ Likewise, you can derive a soft key child using a single `/` after the mnemonic
 ```
+You can inspect password-protected keys either by passing the `--password` flag or using `///` at
+the end of the mnemonic:
 
-The first two levels (`//joe//polkadot`) are hard-derived, while the leaf (`/0`) is soft-derived.
-
-To use key derivation with a password-protected key, add your password to the end:
+_Command (--password flag):_
+```bash
+subkey inspect "caution juice atom organ advance problem want pledge someone senior holiday very" --password "pencil laptop kitchen cutter"
+```
+_Output (--password flag):_
+```text
+Secret phrase `caution juice atom organ advance problem want pledge someone senior holiday very` is account:
+  Secret seed:       0xdfc5d5d5235a37fdc907ee1cb720299f96aeb02f9c7c2fcad7ee8c7bfbd2a4db
+  Public key (hex):  0xdef8f78b123475265815b65a7c55e105e1ab185f4969954f68d92b7bb67a1045
+  Public key (SS58): 5H74SqH1iQCWh5Gumyghh1WJMcmM6TdBHYSK7mKVJbv9NuSK
+  Account ID:        0xdef8f78b123475265815b65a7c55e105e1ab185f4969954f68d92b7bb67a1045
+  SS58 Address:      5H74SqH1iQCWh5Gumyghh1WJMcmM6TdBHYSK7mKVJbv9NuSK
+```
 
-_Command:_
+_Command (password with `///`):_
 ```bash
-# mnemonic phrase plus derivation path plus password
-subkey inspect "image stomach entry drink rice hen abstract moment nature broken gadget flash/joe/polkadot/0///pencil laptop kitchen cutter"
+subkey inspect "caution juice atom organ advance problem want pledge someone senior holiday very///pencil laptop kitchen cutter"
 ```
-_Output:_
+_Output (password with `///`):_
 ```text
-Secret Key URI `image stomach entry drink rice hen abstract moment nature broken gadget flash/joe/polkadot/0///pencil laptop kitchen cutter` is account:
-  Secret seed:      n/a
-  Public key (hex): 0xc69291081743ba7e8f587f1e848d1ffc7b634ce21793a231b7ac75d430b59068
-  Account ID:       0xc69291081743ba7e8f587f1e848d1ffc7b634ce21793a231b7ac75d430b59068
-  SS58 Address:     5GZ4srnepXvdsuNVoxCGyVZd8ScDm4gkGLTKuaGARy9akjTa
+Secret Key URI `caution juice atom organ advance problem want pledge someone senior holiday very///pencil laptop kitchen cutter` is account:
+  Secret seed:       0xdfc5d5d5235a37fdc907ee1cb720299f96aeb02f9c7c2fcad7ee8c7bfbd2a4db
+  Public key (hex):  0xdef8f78b123475265815b65a7c55e105e1ab185f4969954f68d92b7bb67a1045
+  Public key (SS58): 5H74SqH1iQCWh5Gumyghh1WJMcmM6TdBHYSK7mKVJbv9NuSK
+  Account ID:        0xdef8f78b123475265815b65a7c55e105e1ab185f4969954f68d92b7bb67a1045
+  SS58 Address:      5H74SqH1iQCWh5Gumyghh1WJMcmM6TdBHYSK7mKVJbv9NuSK
 ```
+Take note that the `--password` flag _does not_ report the password used, but the `///` _does_. 
+The output is identical otherwise.
+
+## Combine Derivation Paths & Passwords
+
+### IMPORTANT PASSWORD AND DERIVATION NOTE
+
+> Note that the "Secret seed" is the same and _not_ password protected, it can still recover _an_
+> account - but the key pair derived _is not the same_ account as recovered with _any_ password!
+> You can see this in the next section demonstrated.
```

Comment:
```suggestion
> Note that the "secret seed" is the same and _is not_ password protected. Although it can still recover _an_
> account, the key pair that's derived _is not the same_ account as recovered with _any_ password!
> This is demonstrated in the next section.
```